apps/pom.xml

```diff
@@ -88,38 +88,48 @@
 		<sonar.jacoco.reportPath>target/jacoco.exec</sonar.jacoco.reportPath>
 		<sonar.jacoco.itReportPath>target/jacoco-it.exec</sonar.jacoco.itReportPath>
 
-		<metrics.version>4.2.1</metrics.version>
+		<metrics.version>4.2.11</metrics.version>
 		<dropwizard-metrics-newrelic.version>0.7.0</dropwizard-metrics-newrelic.version>
 		<telemetry-http-okhttp.version>0.15.0</telemetry-http-okhttp.version>
-		<aws-java-sdk-bom.version>1.12.276</aws-java-sdk-bom.version>
+		<aws-java-sdk-bom.version>1.12.278</aws-java-sdk-bom.version>
 		<spring-data-jpa.version>2.7.0</spring-data-jpa.version>
 		<!-- Newer (6.0) release versions for hibernate are available for some hibernate components
 		but not all (in particular, entity-manager does not have a 6.0 version available yet).
 		Staying on latest 5.x version until all components have 6.0 versions available. -->
 		<hibernate.version>5.6.10.Final</hibernate.version>
-		<junit.version>5.8.2</junit.version>
+		<junit.version>5.9.0</junit.version>
 		<jackson.version>2.13.3</jackson.version>
-		<mockito.version>4.2.0</mockito.version>
+		<mockito.version>4.6.1</mockito.version>
 		<postgresql.version>42.4.1</postgresql.version>
-		<hsqldb.version>2.6.0</hsqldb.version>
+		<hsqldb.version>2.7.0</hsqldb.version>
 		<slf4j.version>1.7.36</slf4j.version>
 		<hikari.version>5.0.1</hikari.version>
 		<flyway.version>8.5.13</flyway.version>
 		<newrelic.agent.version>7.9.0</newrelic.agent.version>
-		<checkstyle.version>10.1</checkstyle.version>
+		<checkstyle.version>10.3.2</checkstyle.version>
 		<checkstyle.plugin.version>3.1.2</checkstyle.plugin.version>
 		<guava.version>31.1-jre</guava.version>
 		<logback.classic.version>1.2.11</logback.classic.version>
+		<datasource-proxy.version>1.8</datasource-proxy.version>
+		<javapoet.version>1.13.0</javapoet.version>
+		<itextpdf.version>5.5.13.3</itextpdf.version>
+		<auto-service.version>1.0.1</auto-service.version>
+		<zjsonpatch.version>0.4.12</zjsonpatch.version>
+		<hamcrest-library.version>2.2</hamcrest-library.version>
+		<awaitility.version>4.2.0</awaitility.version>
+		<lombok.version>1.18.24</lombok.version>
+		<commons-codec.version>1.15</commons-codec.version>
+
 
 		<!--  Dependencies required for org.apache.commons -->
 		<commons.lang3.version>3.12.0</commons.lang3.version>
-        <commons.csv.version>1.2</commons.csv.version>
-		<commons.io.version>2.7</commons.io.version>
-		<commons.collections4.version>4.1</commons.collections4.version>
+        <commons.csv.version>1.9.0</commons.csv.version>
```

Comment:
can you fix the formatting while you are here?

Reply:
Formatting should be resolved now.